crypto/test/bn_test_lib.c

```diff
@@ -133,7 +133,7 @@ int BN_rand(BIGNUM *rnd, int bits, RAND *rng) {
   bit = (bits - 1) % 8;
   mask = 0xff << (bit + 1);
 
-  buf = OPENSSL_malloc(bytes);
+  buf = (uint8_t*) OPENSSL_malloc(bytes);
```

Comment:
Why did you add this cast? The policy so far is to not cast `(void *)` since C doesn't require the cast, because in general we have a goal of eliminating every single cast.